drivers/pinctrl/pinctrl_eos_s3.c

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2023 Antmicro <www.antmicro.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT quicklogic_eos_s3_pinctrl
+
+#include <zephyr/arch/cpu.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/pinctrl.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/dt-bindings/pinctrl/quicklogic-eos-s3-pinctrl.h>
+#include <soc.h>
+
+LOG_MODULE_REGISTER(pinctrl_eos_s3, CONFIG_PINCTRL_LOG_LEVEL);
```

Comment:
logging not used

Reply:
There's `LOG_ERR`:
https://github.com/zephyrproject-rtos/zephyr/pull/60095/files#diff-267a39e481fb4ee81f32cbd5cb0d7ec45c2cf27a440217d473d451d9dcbb6ea7R98